### ISSUE TITLE
Does not run the map on the mobile device

### ISSUE THREAD

**Issue by EdGh1123** [dmbfm/unity-ar-gps-location-mapbox-route-sample#4]
I try to run the application in the unity environment and the map works, but when I generate the apk the map does not work

here the map is shown (Pc)
![image](https://user-images.githubusercontent.com/54695281/165003185-9490bc95-0fd4-4f58-8cd3-0106793dbb5a.png)

 but not here (Mobile)
![image](https://user-images.githubusercontent.com/54695281/165003269-3f9bc0b3-c70d-486c-9e70-335dd50af511.png)

**Comment by dmbfm** [comment]
Could you check Android's LogCat for any error messages?

Thanks!

**Comment by dmbfm** [comment]
Weird that there aren't any error messages. Which device are you using? Is it correctly connected to the internet? When you try to use the search do you get any results?

Thanks,

**Comment by EdGh1123** [comment]
yes, I have an internet connection, when I do the search I also have results, the device I use is the Xiaomi mi 9t, what settings should I use to start the project, there must be something that I am not doing well

**Comment by dmbfm** [comment]
I can't reproduce the issue here so I can't tell... Have you made any changes to the project or are you using as it is? Have you set the mapbox API token correctly? When you run the app does it request location permissions?

**Comment by EdGh1123** [comment]
I download the project, open it, configure the android 26, activate the Arcore module and run, am I missing a step?
when I install if it asks me for gps and camera permissions

**Comment by dmbfm** [comment]
Send me the project folder in a .zip file to my email, daniel.mbfm@gmail.com so I can take a look.

Thanks,

**Comment by EdGh1123** [comment]
I already sent the project to your email, Thank you very much

**Comment by dmbfm** [comment]
Ok, I think the last commit I just made should fix this. Let me know if it works!
